docs/docs/getting-started/installation/aws.md

```diff
@@ -260,6 +260,16 @@ After setting these parameters, create your `Airy Core` instance with the follow
 airy create --provider aws --provider-config hostUpdate=false
 ```
 
+:::note
+In case you have created your `Airy Core` instance without Let's Encrypt and want to add it later, you must use the `airy upgrade` command.
+
+Even if you don't upgrade to a new version, just modify your `airy.yaml` file as explained in this section and run
+
+`airy upgrade`
+
+After the upgrade is done, continue with setting up your DNS and starting the ingress controller.
+:::
+
```

Comment:
@steffh this PR also makes it possible for Let's Encrypt to be enabled after installation. Please have a look at this doc.
https://deploy-preview-2292--docs-airy.netlify.app/getting-started/installation/aws#https-using-lets-encrypt